<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddProducts</class>
 <widget class="QMainWindow" name="AddProducts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>266</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gestionix - AddProducts</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Product Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="productLine"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Unity price</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="priceBox">
      <property name="prefix">
       <string>$</string>
      </property>
      <property name="suffix">
       <string/>
      </property>
      <property name="maximum">
       <double>999999999999999.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Current Stock</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="stockBox">
      <property name="maximum">
       <number>999999999</number>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../static/resources.qrc"/>
 </resources>
 <connections/>
</ui>
